File: DKR/DKR_1/dkr1_1.py
from math import log10, cos, sin, tan

print('Введите x: ')
x = float(input())

if x < 0:
    if (-7 <= x) and (x < 0):
        if x > 0 and sin(x) != 0:
            y = log10(x) * cos(2 * x) + log10(x) / sin(x)
            print(f'{y:.2f}')
        else:
            print('y не существует')
    else:
        y = 2 - cos(2 * x)
        print(f'{y:.2f}')
else:
    if x != 0 and tan(x) != 0:
        y = (x ** 3 / x ** (1 / 3)) * (x / tan(x))
        print(f'{y:.2f}')
    else:
        print('y не существует')

File: DKR/DKR_1/dkr1_1.pas
program dkr1_1;
var x, y: real;

begin
  writeln('Введите x: ');
  readln(x);
  if x < 0 then begin
    if (-7 <= x) and (x < 0) then begin
      if (x > 0) and (sin(x) <> 0) then begin
        y := log10(x) * cos(2 * x) + log10(x) / sin(x);
        writeln(y:0:2);
      end
      else
        writeln('y не существует');
    end
    else
    begin
      y := 2 - cos(2 * x);
      writeln(y:0:2);
    end;
  end
  else
  begin
    if (x <> 0) and (tan(x) <> 0) then
    begin
      y := (x ** 3 / x ** (1 / 3)) * (x / tan(x));
      writeln(y:0:2);
    end
    else
      writeln('y не существует');
  end;
end.